src/renderer/helpers/api/invidious.js

```diff
@@ -24,7 +24,12 @@ export function invidiousAPICall({ resource, id = '', params = {}, doLogError =
       .then((response) => response.json())
       .then((json) => {
         if (json.error !== undefined) {
-          throw new Error(json.error)
+          // community is empty, no need to display error.
+          if (json.error === 'This channel hasn\'t posted yet') {
```

Comment:
Could you please add a link to the issue on the invidious repo or the one on ours, just so that it's mentioned and the special handling could be removed in the future if the issue is solved.